src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -520,5 +561,52 @@ private void SetBooleanOption(PerLanguageOption<bool> key, int value)
             optionSet = optionSet.WithChangedOption(key, LanguageNames.CSharp, value != 0);
             _optionService.SetOptions(optionSet);
         }
+
+        private static int GetUseVarOption(SimpleCodeStyleOption option)
+        {
+            var offset = option.IsChecked ? 0 : Enum.GetValues(typeof(DiagnosticSeverity)).Length;
+            var baseValue = (int)option.Notification.Value;
+
+            return offset + baseValue;
+        }
+
+        private void SetUseVarOption(Option<SimpleCodeStyleOption> option, int value)
+        {
+            SimpleCodeStyleOption convertedValue = SimpleCodeStyleOption.Default;
+
+            var optionSet = _optionService.GetOptions();
+            switch (value)
+            {
+                case 0:
```

Comment:
Oh yeah, i passionately hate this.  Please encode differently.  it could be a basic string with the two enums encoded in it.